pkgs/tools/misc/redtimer/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, ruby
+, bundlerEnv
+, livesplit-core
+, ncurses
+, toilet
+}:
+
+let
+  gems = bundlerEnv {
+    name = "redtimer-env";
+    gemdir = ./.;
+    inherit ruby;
+  };
+in
+stdenv.mkDerivation rec {
```

Comment:
Is there a reason we cannot use `bundlerApp` here? If so, please add a comment that explains it.